### PR TITLE
Add module guideline

### PR DESCRIPTION
**Pull request by magmukendi-naverz**
-Updated the HowToContribute.md and HowToContribute_KR.md files
- Created a HowToAddModule.md file
- Fixe typos

### CODE REVIEW

File: docs/HowToAddModule.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+In this documentation, you will learn how to make a Zepeto module and add it to the Zepeto Module importer, an Open Source Project. Are you new to Open source projects ? [This guide](https://opensource.guide) is for you.
+
+**Prerequisites:**
+
+- Being able to start a Zepeto Projects
+- Being able to Export a Unity Package
+
+## 1. **Create your Unity Zepeto Project:** 
+    
+You can read more about how to create your Zepeto project by reading the following [guide](https://docs.zepeto.me/studio-world/lang-ko/docs/welcome_zepeto_developers).
```
Comment (jhslee) [inline docs/HowToAddModule.md]: For the URL, can we use the global URL here? `https://docs.zepeto.me/studio-world/docs/welcome_zepeto_developers`

File: docs/HowToAddModule.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    
+## 2. **Add the multiplay feature to your world (optional)**
+You can read more about **ZepetoMultiplay** [here](https://docs.zepeto.me/studio-world/lang-ko/docs/multiplay_tutorial).
```
Comment (jhslee) [inline docs/HowToAddModule.md]: Same thing, please utilize the global language link for reference links.

File: docs/HowToAddModule.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    
+As this is a Zepeto Project, make sure to create your script the Zepeto-way: **Right click** on your project tab, `Create > Zepeto > Typescript`. This will generate a Typescript code (Yes, Zepeto script is written in Typescript and not C-sharp). If you are just getting started with Typescript, your can check out our collection of Interesting Typescript content [here](https://www.typescriptlang.org/docs/). 
+
+![image](https://user-images.githubusercontent.com/131629767/235897562-11cd3677-af54-422c-a5b2-d05570ec6c75.png)
+
+## 5. **Exporting your Unity Package**
+    
+If you have never exported a Unity Package, Check out this [article](https://docs.unity3d.com/2018.1/Documentation/Manual/HOWTO-exportpackage.html) and come back later.
+    
+Already back ? Good! Now make sure that your imported Unity package follows our naming convention v[versionNumber].unitypackage. *(e.g v1.0.1.unitypackage).* If not, don’t worry it;s not too late, you can just rename it. Remember where you save that UnityPackage it will serve us later on this tutorial.
```
Comment (jhslee) [inline docs/HowToAddModule.md]: it;s -> It's

File: docs/HowToAddModule.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+- Within the ***moduleName*** folder, create a **docs** folder where you can add any `README.md` file for the module documentation if there is any. If you have translated version of your `.md` file, make sure to name them as follow: `README_{language}.md`. E.g: README_KR.md.
+- Within the **docs** folder, you can also add an **images** folder which could be used for the visual explanation of your application.
+
+## 8. **Finally Adding your unity package to the Zepeto Module Importer**
+   
+Remember the the Unity Package (**.unitypackage)* you made in step 5 ? Copy and paste it in the `Release > moduleName` folder.
+   
+## 9. **Push to your github**
+ 
+Before doing the magic push to github, make sure that your module and its elements respect all the naming convention I told you about in this guide. If everything looks, feel free to push to you github. You know how it works right ? `git add .` , `git commit -m "[insert your commit message here]` and finally `git push`.
```
Comment (jhslee) [inline docs/HowToAddModule.md:79]: Can we link a github tutorial guide link here? I think that would help newer users that don't understand git

File: docs/HowToContribute.md [hunk 5]
```diff
@@ -38,28 +38,4 @@ Please suggest new features or improvements! You can submit a feature request by
 5. Push your changes to your GitHub fork.
```
Comment (jhslee) [inline docs/HowToContribute.md:40]: I wonder if we can also add here a link to how to make a fork and clone from an existing repository?

File: docs/HowToAddModule.md (hunk 1 above)
Comment (magmukendi-naverz) [reply docs/HowToAddModule.md]: Done sir :)

File: docs/HowToAddModule.md (hunk 2 above)
Comment (magmukendi-naverz) [reply docs/HowToAddModule.md]: Done :)

File: docs/HowToAddModule.md (hunk 3 above)
Comment (magmukendi-naverz) [reply docs/HowToAddModule.md]: Thanks for noticing this typo.

File: docs/HowToAddModule.md (hunk 4 above)
Comment (magmukendi-naverz) [reply docs/HowToAddModule.md:79]: I added a link to a github table of contents for people just getting started.

File: docs/HowToContribute.md (hunk 5 above)
Comment (magmukendi-naverz) [reply docs/HowToContribute.md:40]: Done sir :)

**Pull request merged**